CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.39.1](https://github.com/buildkite/agent/tree/v3.39.1) (2022-10-14)
+[Full Changelog](https://github.com/buildkite/agent/compare/v3.39.0...v3.39.1)
+
+### Security
+- Replace use of export -p with a dedicated agent command for printing environment [#1781](https://github.com/buildkite/agent/pull/1781) (@moskyb)
```

Comment:
i think we should be really clear about the security impact here rather than the fix - i don't think users really care that we fix it by doing xyz, they care about the fact that there was a security issue.

perhaps something like
```suggestion
- Fix security issue where in certain situations, agents using bash v5.2 could have some environment variables leak to job logs. See the [issue](https://github.com/buildkite/agent/issues/1780) and [our security advisory](https://buildkite.com/changelog/167-security-advisory-for-buildkite-agent-and-bash-5-dot-2) for more information. [#1781](https://github.com/buildkite/agent/pull/1781) (@moskyb)
```

also, credit yourself! 
```suggestion
- Replace use of export -p with a dedicated agent command for printing environment [#1781](https://github.com/buildkite/agent/pull/1781) (@moskyb + @triarius + @pda)
```